game.go

```diff
@@ -498,10 +500,11 @@ func (p *Game) loadIndex(g reflect.Value, index interface{}) (err error) {
 	} else {
 		p.baseObj.initWithSize(proj.Map.Width, proj.Map.Height)
 	}
```

Comment:
value of `worldWidth_/worldHeight_`:
* if user specifies map.width and map.height, then: worldWidth_, worldHeight_ = map.width, map.height.
* if not, then: worldWidth_, worldHeight_ = float64(sceneImg.width) / map.gridUnit, float64(sceneImg.height) / map.gridUnit.

Reply:
value of `windowWidth_/windowHeight_`:

* width, height := int(worldWidth_ * map.gridUnit), int(worldHeight_ * map.gridUnit)
* if user specifies conf.width and conf.height, then: windowWidth_, windowHeight = conf.width, conf.height (if larger then width/height, limit it).
* if not, then: windowWidth_, windowHeight = width, height (if out of screen size, limit it).